04_model_inference.ipynb

```diff
@@ -0,0 +1,2792 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "3f5d1031-4a3c-4bd7-9089-4766917156f2",
+   "metadata": {},
+   "source": [
+    "# Run inference on time to merge model trained previously\n",
+    "\n",
+    "\n",
+    "## What we did previously\n",
+    "\n",
+    "In the previous [notebook](./03_model_training.ipynb) we trained machine learning models to classify a PR's `time_to_merge` into one of the 10 bins (or \"classes\"). We then deployed the model with the highest f1-score as a service using the model saved in s3.\n",
+    "\n",
+    "## In this step\n",
+    "\n",
+    "\n",
+    "The purpose of this notebook is to check whether this service is running as intended, and more specifically to ensure that the model performance is what we expect it to be. So here, we will use the test set from the aforementioned notebook as the query payload for the service, and then verify that the return values are the same as those obtained during training/testing locally.\n",
```

Comment:
The model is not a service. Lets say that we are sending the payload to the saved model on S3 to see how the model is performing on the test data ( a segment of past reviews )